extras/rya.export/export.api/src/main/java/org/apache/rya/export/api/conf/MergeConfiguration.java

```diff
@@ -55,7 +56,10 @@
     private final int childPort;
 
     private final MergePolicy mergePolicy;
+    private final CopyType copyType;
```

Comment:
remove this, we can talk during scrum about keeping export/import path (pick a naming convention input/output or import export)